.github/workflows/lint.yml

```diff
@@ -1,44 +1,25 @@
-name: Lint
+name: lint
 on: push
 
 defaults:
   run:
     shell: bash
 
 jobs:
-  deps:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-      - name: Install Go
-        uses: actions/setup-go@v3
-        with:
-          go-version: 1.19.x
-      - name: Check dependencies
-        run: |
-          go version
-          test -z "$(go mod tidy  && git status --porcelain)"
-          go mod verify
-
   lint:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-      - name: Install Go
-        uses: actions/setup-go@v3
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v4
         with:
-          go-version: 1.19.x
+          go-version: 1.20.x
+
       - name: Retrieve golangci-lint version
         run: |
           echo "::set-output name=Version::$(head -n 1 "${GITHUB_WORKSPACE}/.golangci.yml" | tr -d '# ')"
         id: version
-      - name: golangci-lint
+
+      - name: Run golangci-lint
```

Comment:
Is the golangci.yml file config up to date?

Reply:
Actually, I thought it wasn't, but turns out that it is already up to date 👍🏻